bot/tasks/pennychat.py

```diff
@@ -73,59 +76,100 @@ def share_penny_chat_invitation(penny_chat_id):
     penny_chat_invitation.save()
 
 
-def datetime_template(penny_chat):
-    timestamp = int(penny_chat.date.astimezone(utc).timestamp())
-    date_text = f'<!date^{timestamp}^{{date_pretty}} at {{time}}|{penny_chat.date}>'
-    return date_text
+def send_penny_chat_reminders():
+    """This sends out reminders for any chat that is about to happen."""
+    slack_client = _get_slack_client()
```

Comment:
Why doesn't this have the `@background` decorator? I'm confused as to how this is processed.

Reply:
Since I _don't_ `@background` this function then (as you are thinking) it is NOT run in the background using that new framework. Rather I'm just sticking related functions into the same file.